src/translations/nbTranslations.json

```
@@ -432,6 +433,7 @@
     },
     "success": {
       "add_employment": "Ansettelse lagt til",
+      "add_promoted_publication": "Resultatet ble markert som favoritt",
```

Ser at denne brukes om man fjerner en favoritt også. usikker på hva som er en god tekst her da 😅 
```suggestion
      "updated_promoted_publication": "Favorittmarkering ble oppdatert",
```